src/DataTable/BulkActions.jsx

```diff
@@ -6,7 +6,10 @@ import DataTableContext from './DataTableContext';
 import Actions from './CollapsibleButtonGroup';
 
 const BulkActions = ({ className }) => {
-  const { bulkActions } = useContext(DataTableContext);
+  const { bulkActions, selectedFlatRows } = useContext(DataTableContext);
+  if (typeof bulkActions === 'function') {
```

Comment:
Not a fan of using this method to distinguish these, I think one alternative might be using a new prop, `customBulkActions` that would override `bulkActtions`. Instead of just using a single prop to service both types of params.

Reply:
I agree, a new prop that takes precedence seems preferable.